bottlerocket/tests/workload/nvidia-smoke.yaml

```diff
@@ -0,0 +1,16 @@
+kind: Test
+metadata:
+  name: workload-agent
+  namespace: testsys
+spec:
+  agent:
+    name: workload-agent
+    image: __WORKLOAD_AGENT_IMAGE__
```

Comment:
Is this a templating mechanism?

Reply:
In most of the documentation I have written I have used `<SCREAMING-KEBAB-CASE>` to identify a value that should be replaced.

Reply:
I've changed this to match that pattern.